src/App.vue

```diff
@@ -470,41 +470,17 @@ export default {
 
     storeTasks (response: any): void {
       const tasks = response?.data?.tasks
-      if (this.businessId) {
-        if (tasks) {
-          this.setTasks(tasks)
-        } else {
-          throw new Error('Invalid tasks')
-        }
-      }
-      // FUTURE: update this when API returns New Incorporation task (#3102)
-      if (this.nrNumber) {
-        if (tasks) {
-          // if we have existing tasks, use them
-          if (tasks.length > 0) {
-            this.setTasks(tasks)
-            this.setEntityStatus(EntityStatus.INCORPORATION_APPLICATION)
-          } else {
-            // otherwise create a New Incorporation task
-            tasks.push({
-              enabled: true,
-              order: 1,
-              task: {
-                todo: {
-                  nameRequest: this.nameRequest,
-                  header: {
-                    name: FilingTypes.NAME_REQUEST,
-                    status: FilingStatus.NEW
-                  }
-                }
-              }
-            })
-            this.setTasks(tasks)
+      if (tasks) {
+        this.setTasks(tasks)
+        if (this.nrNumber && tasks.length > 0) {
+          if (tasks[0].task.todo) {
```

Comment:
Should be
`tasks[0].task?.todo`

Reply:
Could you also please remove the tasks and filings workarounds on lines 462-468 and 492-498?

Reply:
Done